snap/snapcraft.yaml

```diff
@@ -27,8 +27,8 @@ parts:
   eclipse:
     plugin: dump
     source:
-      - on amd64: http://ftp.halifax.rwth-aachen.de/eclipse/technology/epp/downloads/release/photon/$SNAPCRAFT_PROJECT_VERSION/eclipse-java-photon-$SNAPCRAFT_PROJECT_VERSION-linux-gtk-x86_64.tar.gz
-      - on i386: http://ftp.halifax.rwth-aachen.de/eclipse/technology/epp/downloads/release/photon/$SNAPCRAFT_PROJECT_VERSION/eclipse-java-photon-$SNAPCRAFT_PROJECT_VERSION-linux-gtk.tar.gz
+      - on amd64: http://ftp.halifax.rwth-aachen.de/eclipse/technology/epp/downloads/release/$SNAPCRAFT_PROJECT_VERSION/R/eclipse-java-$SNAPCRAFT_PROJECT_VERSION-linux-gtk-x86_64.tar.gz
+      - on i386: http://ftp.halifax.rwth-aachen.de/eclipse/technology/epp/downloads/release/$SNAPCRAFT_PROJECT_VERSION/R/eclipse-java-$SNAPCRAFT_PROJECT_VERSION-linux-gtk.tar.gz
```

Comment:
The build is failing in travis because it's taking the R from version and the R in the url and making /R/R. So as mentioned above, leave this line as is, and change the version line and we should be good!